src/sentry/api/invite_helper.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+
+from django.utils.crypto import constant_time_compare
+from django.core.urlresolvers import reverse
+
+from sentry.models import AuditLogEntryEvent, Authenticator, OrganizationMember
+from sentry.signals import member_joined
+
+PENDING_INVITE = 'pending-invite'
+COOKIE_MAX_AGE = 60 * 60 * 24 * 7  # 7 days
+
+
+class ApiInviteHelper(object):
+    def __init__(self, instance, request, member_id, token, logger=None):
+        self.request = request
+        self.instance = instance
+        self.member_id = member_id
+        self.token = token
+        self.logger = logger
+        self.om = self.get_organization_member()
+
+    def handle_success(self):
+        member_joined.send_robust(
+            member=self.om, organization=self.om.organization, sender=self.instance)
+
+        pass
```

Comment:
You don't need this as the method has a body.

Reply:
Whoops leftover